### ISSUE TITLE
Potential bug when creating custom functionality in PickerActivity

### ISSUE THREAD

**Issue by ovidiurosu** [Flipboard/bottomsheet#62]
Hi all,

I've found the following potential bug when creating a custom Activity for the PickerActivity.class from bottomsheet-sample.

I have the following basic implementation for the custom functionality:

    Drawable scheduledGreetingDrawable = ResourcesCompat.getDrawable(getResources(), R.mipmap.ic_launcher, null);
                IntentPickerSheetView.ActivityInfo scheduledGreetingInfo =
                        new IntentPickerSheetView.ActivityInfo(scheduledGreetingDrawable, getString(R.string.scheduled), CreateScheduledGreetingActivity.class);
    intentPickerSheet.setMixins(Collections.singletonList(scheduledGreetingInfo));

Now `CreateScheduledGreetingActivity` is in the following package: `com.deolabs.greetingsbook.activities`

I inspect the concreteIntent from `void onIntentPicked(IntentPickerSheetView.ActivityInfo activityInfo)`: 

`Intent concreteIntent = activityInfo.getConcreteIntent(shareIntent);`

This issue is that the value from mComponent->mPackage is `com.deolabs.greetingsbook.activities`
A normal intent has the value of mPackage `com.deolabs.greetingsbook` and works fine.

Basically if I move my activity in the the `com.deolabs.greetings' root package it works fine. Should'nt I be able to invoke activities from any custom package from the root package ? The bug is fixed if the activity class is placed in the root package (com.deolabs.greetings). If I put it in (com.deolabs.greetings.activities) it crashes.

The Exception that is thrown is:

    Unable to find explicit activity class     {com.deolabs.greetingsbook.activities/com.deolabs.greetingsbook.activities.CreateScheduledGreetingActivity}; have you declared this activity in your AndroidManifest.xml?

Thanks.

**Comment by markrietveld** [comment]
If you move an Activity into another package, you also need to update AndroidManifest.xml with the new location. That's what the error message says: "have you declared this activity in your AndroidManifest.xml?". This is not related to bottomsheet

**Comment by ovidiurosu** [comment]
![bug](https://cloud.githubusercontent.com/assets/3541136/9839369/85294c0c-5a80-11e5-919e-47c6a214f50c.png)

Yes, it was updated. I've added a screenshot.

In order to test it just create a package in the bottomsheet-sample project and move MainActivity inside it. As you can see AndroidManifest.xml is updated.

The run the Intent Picker, write some text and select the custom functionality 'Custom mix-in'. It shouldnt crash. Tested on a Nexus 7 tablet.

I'm running the latest Android M version, could be on their side then.

**Comment by markrietveld** [comment]
Yes, you're right. I followed the steps you put it the sample, and got the same issue. It appears we need to fix the `getConcreteIntent` method

**Comment by ovidiurosu** [comment]
Great then. Hope it gets fixed :) Good job, with the project also.

**Comment by markrietveld** [comment]
@ovidiurosu You should be able to use version 1.4.3 now, which fixes this issue.

**Comment by ovidiurosu** [comment]
@markrietveld Great job, thanks for the support.
